inclusive-design/wecount.inclusivedesign.ca
# Search locally

* [X] This pull request has been tested by running `npm run build` and reviewing generated site

## Description

Resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/214.

Perform the search by fetch pre-generated data from `{origin-url}/index.json` rather than using the WordPress search API. It removes the dependency on the backend WordPress completely once the website is built.

One issue I noticed is when a change to data-fetcher requires `index.json` to re-populate but `index.json` already exists, `index.json` is not automatically re-populated with the new data. This issue can be fixed by manually removing `index.json` before rebuild. I'm not sure if the problem will happen with Netlify. Let's keep an eye on it.

## Steps to test

1. Perform a search;
2. Watch the "network" tab on the developer tool;
3. There shouldn't have any call to the WordPress API;
4. Search results should be displayed in the proper grid layout.
